arm-customer-insights/2017-01-01/swagger/customer-insights.json

```diff
@@ -5306,6 +5479,89 @@
         "modelAsString": false
       },
       "description": "Supported permission types."
+    },
+    "ProvisioningState": {
+      "type": "string",
+      "enum": [
+        "Provisioning",
+        "Succeeded",
+        "Expiring",
+        "Deleting",
+        "HumanIntervention",
+        "Failed"
+      ],
+      "x-ms-enum": {
+        "name": "ProvisioningStates",
+        "modelAsString": true
+      },
+      "description": "Provisioning state."
+    },
+    "ConnectorType": {
+      "type": "string",
+      "enum": [
+        "None",
+        "CRM",
+        "AzureBlob",
+        "Salesforce",
+        "ExchangeOnline",
+        "Outbound"
+      ],
+      "x-ms-enum": {
+        "name": "ConnectorTypes",
+        "modelAsString": true
+      },
+      "description": "Type of connector."
+    },
+    "SuggestRelationshipLinksResponse": {
+      "required": [
+        "interactionName",
+        "suggestedRelationships"
+      ],
+      "properties": {
+        "interactionName": {
+          "type": "string",
+          "description": "The interaction name."
+        },
+        "suggestedRelationships": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/RelationshipsLookup"
+          },
+          "description": "Suggested relationships for the type."
+        }
+      },
+      "description": "The response of suggest relationship links operation."
+    },
+    "RelationshipsLookup": {
```

Comment:
Used above, so same question. If you plan to use this type in requests, too, properties should of course stay writable